pkg/crd/gen.go

```diff
@@ -206,9 +206,9 @@ func FindMetav1(roots []*loader.Package) *loader.Package {
 // FindKubeKinds locates all types that contain TypeMeta and ObjectMeta
 // (and thus may be a Kubernetes object), and returns the corresponding
 // group-kinds.
-func FindKubeKinds(parser *Parser, metav1Pkg *loader.Package) []schema.GroupKind {
+func FindKubeKinds(parser *Parser, metav1Pkg *loader.Package) map[schema.GroupKind]bool {
 	// TODO(directxman12): technically, we should be finding metav1 per-package
-	var kubeKinds []schema.GroupKind
+	kubeKinds := map[schema.GroupKind]bool{}
```

Comment:
Why the map with key= GroupKind and value=bool to say it was or not found it required. 
Should not more make sense return a list with all found GroupKinds?

Reply:
`bool` was intended be a placeholder.
I have replaced it with an empty struct.

Reply:
Map helps de-duplicating group-kind

Reply:
Tks for the clarification.